types/nightwatch/index.d.ts

```diff
@@ -4,8 +4,10 @@
 //                 Connor Schlesiger <https://github.com/schlesiger>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export interface NightwatchCustomPageObjects {
-    page: {};
+/* tslint:disable:max-line-length */
+
+export interface NightWatchCustomPageObjects {
+    page: {[name: string]: () => NightWatchBrowser};
```

Comment:
Now that I've looked at the source code, I'm not sure this is the correct interface for Nightwatch Page Objects (https://github.com/nightwatchjs/nightwatch/wiki/Page-Object-API).

Reply:
To be honest I am not all that familiar with Nightwatch. I can look at any additions to your PR and take your word for it. If you'd like to become a maintainer of this definition I will approve that as well.

Reply:
I have no intention of becoming maintainer of this definition, but I will gladly revise this PR to more accurately reflect the actual JS interface.